string_manipulation.rb

```diff
@@ -38,9 +37,14 @@ def reverse_words(my_words)
   return my_words
 end
 
+
+# sentence = "Yoda  is   awesome"
+# puts "Original: #{sentence}"
+# reversed_sentence = "awesome   is  Yoda"
 # A method to reverse the words in a sentence, in place.
 def reverse_sentence(my_sentence)

```

Comment:
Once you update reverse_words to have a space complexity of O(1), the space complexity of this method will also become O(1).